test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -148,13 +148,13 @@ def _get_latest_package_version(package):
 @pytest.mark.model("N/A")
 @pytest.mark.canary("Run safety tests regularly on production images")
 @pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
-@pytest.mark.skipif(
-    not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
-    reason=(
-        "Skipping the test to decrease the number of calls to the Safety Check DB. "
-        "Test will be executed in the 'mainline' pipeline and canaries pipeline."
-    )
-)
+# @pytest.mark.skipif(
```

Comment:
things which are added to configs and this commented change will be revert

Reply:
@jeet4320 can you clarify what you mean?

Reply:
before merging this PR, changes added to  dlc_developer_config.toml need to be reverted, which I'll do